### PR TITLE
Add CVE-2017-15089

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: database/java/2017/15089.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+cve: 2017-15089
+title: CVE in org.infinispan:infinispan-client-hotrod
+description: >
+    It was found that the Hotrod client in Infinispan before 9.2.0.CR1 would unsafely read deserialized data on information from the cache. An authenticated attacker could inject a malicious object into the data cache and attain deserialization on the client, and possibly conduct further attacks.
+cvss_v2: 6.5
+references:
+    - http://www.securitytracker.com/id/1040360
+    - https://access.redhat.com/errata/RHSA-2018:0294
```
Comment (GeetikaBatra) [inline database/java/2017/15089.yaml]: This link looks wrong as it contains different information than the package, it talks about `Red Hat JBoss Data Grid`, while the dependency is `org.infinispan:infinispan-client-hotrod`

**Comment by GeetikaBatra** [comment]
@msrb, PTAL

File: database/java/2017/15089.yaml (hunk 1 above)
Comment (msrb) [reply database/java/2017/15089.yaml]: Good catch :) It's actually fine as Red Hat JBoss Data Grid is a RH product based on Infinispan (upstream name).

File: database/java/2017/15089.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - https://github.com/infinispan/infinispan/pull/5639
+
+affected:
+    - groupId: org.infinispan
+      artifactId: infinispan-client-hotrod
+      version:
+        - "<=9.2.0.CR1"
```
Comment (msrb) [inline database/java/2017/15089.yaml]: This is actually the first version which contains a fix for this vulnerability. Previous (affected) version is [`9.2.0.Beta2`](https://mvnrepository.com/artifact/org.infinispan/infinispan-client-hotrod) so this should be `"<=9.2.0.Beta2"`.

**Comment by GeetikaBatra** [comment]
@msrb  PTAL :)

**Pull request closed without merging**